### PR TITLE
Add code skeleton

### PR DESCRIPTION
**Pull request by ErikRichardS**
Solves Issue #20 and adds a code skeleton for the main.java file that includes the global variable declarations and a DECIDE function without any functionality. Is a java version of C code given in the assignment PDF.

### CODE REVIEW

File: src/main/Main.java [hunk 1]
```diff
@@ -1,9 +1,94 @@
+import java.lang.Math;
 
 public class Main {
 	
+	// CONSTANT
```
Comment (nilsstre) [inline src/main/Main.java:5]: We should place them in a separate class file only containing the constants. So it's easy to import them to the classes that need them.

**Review by nilsstre — CHANGES_REQUESTED** [review]

File: src/main/Main.java [hunk 2]
```diff
@@ -1,9 +1,94 @@
...
+	public static boolean[][] PUM;
+	
+	// Final Unlocking Vector
+	public static boolean[] FUV;
+	
+	// Decision: Launch or No Launch
+	public static boolean LAUNCH;
+	
+	// Compares floating point numbers
+	public static COMPTYPE DOUBLECOMPARE(double A, double B) {
```
Comment (nilsstre) [inline src/main/Main.java:68]: I believe that we should use BigDecimal for floating points arithmetics in Java. See: [link](https://stackoverflow.com/a/179453)

**Review by nilsstre — CHANGES_REQUESTED** [review]

File: src/main/Main.java [hunk 3]
```diff
@@ -1,9 +1,94 @@
+import java.lang.Math;
 
 public class Main {
 	
+	// CONSTANT
+	public static final double PI = 3.1415926535;
```
Comment (nilsstre) [inline src/main/Main.java:6]: I think that we should use `Math.PI` instead.

**Comment by nilsstre** [comment]
Is this a direct copy of the C code in the instructions? We should change it before we merge it with master.

File: src/main/Main.java (hunk 1 above)
Comment (ErikRichardS) [reply src/main/Main.java:5]: There is only one constant and that is PI. The rest are variable declarations. You want to move all of them to a separate file?

**Comment by ErikRichardS** [comment]
> Is this a direct copy of the C code in the instructions? We should change it before we merge it with master.

It's not a direct copy. I tried to cut down the unnecessary parts as best I could as the C code has a lot of typedefs and the like that I figured wouldn't be necessary in Java.

File: src/main/Main.java (hunk 2 above)
Comment (ErikRichardS) [reply src/main/Main.java:68]: That is gonna make the code a lot more "heavy" compared to using regular arithmetic symbols.

**Comment by nilsstre** [comment]
I have made a revision of the code skeleton for the project based on this branch. I have created a new pull request it and is therefor closing this pull request. See pull request #30 for more information.

**Pull request closed without merging**